install_config/persistent_storage/persistent_storage_ceph_rbd.adoc

```diff
@@ -34,27 +40,71 @@ storage provider.
 ====
 
 [[ceph-provisioning]]
-
 == Provisioning
+
 Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision Ceph volumes in OpenShift, the
-following are required:
+a volume in OpenShift. To provision Ceph volumes in OpenShift, the following
+information is required:
 
-- Existing Ceph volumes, to be defined in the persistent volume object
-- The `*PersistentVolume*` API
+- The ceph key (in base64) to be used as an OpenShift secret object.
```

Comment:
@jeffvance the API of the ceph volume is in terms of the name of the secret to refer to, the ceph plugin knows nothing about how secret data is encoded when serialized.